### ISSUE TITLE
any Touch related must be separate

### ISSUE THREAD

**Issue by katopz** [Gamua/Starling-Framework#96]
IMHO any touch related must have it own  library separated from Starling
I'm saying this because i did separate it and use it with normal 2D app (without Stage3D need)
the pro for this is i do save my plenty of time to test touch on real device and use fake touch from Starling via simulator instead
which mean it can be separate for no need touch app for smaller size and/or even can be use with normal 2D app

In short
Please separate Touch related as another library so it can be use with other app with/with out Stage3D need
